com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -530,7 +530,7 @@ private void Awake()
             }
         }
 
-        private void Update()
+        public virtual void Update()
```

Comment:
suggest to make these protected virtual so we aren't allowing outside classes to call them manually